src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+# Powered Hardsuits (not modsuits)
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| Alzore_ (Discord) | :x: No | TBD |
+
+## Overview
+
+Implementation of powered hardsuits to the game, and various NON-INTERCHANGEABLE FEATURES to specific hardsuits for them to better excel at their given job. Hardsuits would not be able to compress, they are not modsuits.
+
+## Background
+
+In Space Station 13 (or at least on TG), hardsuits basically don't exist and are replaced with the atrocity known as modsuits. These suits do the following:
+- are extremely portable
+- have little to no downsides to wear
+- can be easily crafted
+- can be loaded up with all sorts of meta equipment
+- require power
+- are basically all the same (due to "modularity" allowing you to put the meta modules in every single suit)
+
+Hardsuits in Space Station 14 are effectively the polar opposite of this, except for the last point. They are all mostly the same, with the only difference between suits being movement speed and armor. What if that wasn't the case though?
+
+## Hardsuits, EVA, and Void suits
+
+Before I get into the cool ideas of what different spacesuits would do, there should be a few specific classifications for spacesuits, since not everything needs abilities.
+
+### EVA suits
+EVA suits would be an abundant and renewable (through cargo) form of space protection. The helmet comes separated and needs to be attached and carried on its own. They provide absolutely zero combat damage resistance. Movement speed and cold protection vary.
+This basically only includes EVA suits, emergency EVA suits, and prisoner EVA suits.
+EVA suits with resistances would be counted as voidsuits.
+
+### Voidsuits
+Voidsuits would basically function as hardsuits without the fancy tech. They would have no abilities, decent movement speed, and not require power. People such as the captain, paramedic, and CMO would get these. The atmos firesuit would sorta fall under this category despite not being called a voidsuit.
+Voidsuits can still have headlamps in the helmet, but these would either use power cells or just have infinite power.
```

Comment:
I know this is just going to turn into slllllow powercreep... but maybe we can have very, extremely small benefits for some void suits. Spitballing here. Im not entirely sure what benefits the para's suit would get besides a health scanner/hud (maybe further when Newmed happens).

Half-related, the CMOs suit is actually a hardsuit, so maybe you could integrate a health scanner(?), point light, MediHUD, etc into it.  This would actually incentivize its use while also making it so that it's just unviable to wear constantly (as right now CMO just gets a free 10% slowdown no armor free pass for spacing with zero downsides, they're free to wear it anytime)